### PR TITLE
[CHK-918] (branch 914) transaction sul refactoring della libreria commons

### PR DESCRIPTION
**Pull request by ciuffagianluca**
Depends on pagopa/pagopa-ecommerce-commons#15

After commons library update, also transaction need to be aligned with this new version.

#### List of Changes

- Transaction now fit new ecommerce commons, in details now the `NoticeCode` name is `PaymentNotice` (and all get methods that return this kind of data changed their name)
- fee total in transactioninfo is not filled if it is null in transaction document 
- Amount total removed from transactionInfo

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: pom.xml
Comment (ciuffagianluca) [inline pom.xml:519]: Before merge it, please update with new ecommerce-commons version

Comment (giovanniberti) [reply pom.xml:519]: Added `do not merge` label to reflect this

**Review by giovanniberti — APPROVED** [review]

**Pull request merged**
